src/core/fonts.js

```diff
@@ -2050,35 +2050,45 @@ var Font = (function FontClosure() {
         var oldGlyfData = glyf.data;
         var oldGlyfDataLength = oldGlyfData.length;
         var newGlyfData = new Uint8Array(oldGlyfDataLength);
-        var startOffset = itemDecode(locaData, 0);
-        var writeOffset = 0;
+
+        // The spec says the offsets should be in ascending order, however
+        // this is not true for some fonts or they use the offset of 0 to mark a
+        // glyph as missing. OTS requires the offsets in order and not to
```

Comment:
Nit: Looks like a missing "to be" here, i.e. it probably should read:
`// glyph as missing. OTS requires the offsets to be in order and not to`